e2e_tests/tests/views/test_group.py

```diff
@@ -0,0 +1,88 @@
+"""Test functions related to sample groups."""
+
+import pytest
+from pathlib import Path
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.wait import WebDriverWait
+
+from ..conftest import get_bootstrap_alert, get_element_by_test_id
+
+
+@pytest.mark.parametrize("group_id", ["mtuberculosis", "saureus", "ecoli"])
+def test_open_group_view(logged_in_user, config, group_id: str):
+    """Test that the test groups are working."""
+
+    # FIRST goto the group view
+    url = Path(config["frontend_url"]) / 'groups' / group_id
+    logged_in_user.get(str(url))
+
+    # TEST that the page could load
+    assert "bonsai" in logged_in_user.title.lower()
```

Comment:
is a http code == 200 test a good idea here?

Reply:
Selenium doesnt expose the HTTP response code by default. People have been creating workaround like this for that purpose, https://gist.github.com/LeMoussel/bfed69044cb65947a4e63c76f11e6e1f.

Reply:
I think that is something we could add in a followup PR perhaps as the tests grows.